apps/api/src/app/routes/__chainId/yield/getPoolsInfo.ts

```diff
@@ -0,0 +1,58 @@
+import { FastifyPluginAsync } from 'fastify';
+import { FromSchema } from 'json-schema-to-ts';
+import { PoolInfo } from '../../../data/poolInfo';
+import { In } from 'typeorm';
+import { poolsInfoBodySchema, errorSchema, paramsSchema, poolsInfoSuccessSchema } from './schemas';
+import { POOLS_QUERY_CACHE, POOLS_RESULT_LIMIT } from './const';
+import { trimDoubleQuotes } from './utils';
+
+type RouteSchema = FromSchema<typeof paramsSchema>;
+type SuccessSchema = FromSchema<typeof poolsInfoSuccessSchema>;
+type ErrorSchema = FromSchema<typeof errorSchema>;
+type BodySchema = FromSchema<typeof poolsInfoBodySchema>;
+
+const root: FastifyPluginAsync = async (fastify): Promise<void> => {
+  fastify.post<{
+    Params: RouteSchema;
+    Reply: SuccessSchema | ErrorSchema;
+    Body: BodySchema;
+  }>(
+    '/getPoolsInfo',
```

Comment:
This method and path is not following very match the restful structure
i.e. https://restfulapi.net/resource-naming

what about `GET /pools/OxB7ECB2AA52AA64A717180E030241BC75CD946726`

Reply:
Agree, I just inhertted this style from the existing paths:
<img width="441" alt="image" src="https://github.com/user-attachments/assets/322a9e1d-9b86-4b4f-bc05-d9b56246c1fc">

Anyway, I will rename it

Reply:
Renamed

<img width="477" alt="image" src="https://github.com/user-attachments/assets/44aeb7d3-ea3a-4965-ac69-4135850c8e83">